tests/components/twinkly/test_light.py

```diff
@@ -171,6 +171,110 @@ async def test_turn_on_with_effect(hass: HomeAssistant):
     assert client.default_mode == "movie"
 
 
+async def test_turn_on_with_color_rgbw_and_missing_effect(hass: HomeAssistant):
+    """Test support of the light.turn_on service with rgbw color and missing effect support."""
+    client = ClientMock()
+    client.state = False
+    client.device_info["led_profile"] = "RGBW"
+    client.brightness = {"mode": "enabled", "value": 255}
+    entity, _, _, _ = await _create_entries(hass, client)
+
+    assert hass.states.get(entity.entity_id).state == "off"
+    assert not client.current_movie
+
+    assert LightEntityFeature.EFFECT in entity.supported_features
+    client.mock_unsupported_effects = True
+
+    # Turn off to force async_update
+    await hass.services.async_call(
+        "light", "turn_off", service_data={"entity_id": entity.entity_id}
+    )
+    await hass.async_block_till_done()
```

Comment:
```suggestion
    await hass.services.async_call(
        "light", "turn_off", service_data={"entity_id": entity.entity_id}, blocking=True
    )
```

Reply:
Thank you.  Updated all the tests to use this.